requirements.txt

```diff
@@ -7,6 +8,5 @@ openai==0.27.8
 python-multipart==0.0.6
 fastapi==0.97.0
 uvicorn==0.22.0
-python-dotenv==1.0.0
+python-dotenv
```

Comment:
please add the version here

Reply:
in some hosting platform i got the error of not finding python-dotenv==1.0.0 however is the correct version I have, so I got rid of it :)

Reply:
ok, it's pretty weird